### PR TITLE
[Refactoring] Small constant refactoring in DomainQuotaRoutesTest

### PR DESCRIPTION
**Pull request by Arsnael**
Let's stop using french constants with accents and replace them by their english equivalent ones instead.

It's more cosmetic than anything else, but it has been bugging me for a while now :)

### CODE REVIEW

File: server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/DomainQuotaRoutesTest.java [hunk 1]
```diff
@@ -43,16 +43,16 @@
 class DomainQuotaRoutesTest {
 
     private static final String QUOTA_DOMAINS = "/quota/domains";
-    private static final String PERDU_COM = "perdu.com";
-    private static final Domain TROUVÉ_COM = Domain.of("trouvé.com");
+    private static final String LOST_COM = "lost.com";
+    private static final Domain FOUND_COM = Domain.of("found.com");
```
Comment (rouazana) [inline server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/DomainQuotaRoutesTest.java]: trouvé.com and perdu.com are real very simple html site, while found.com and lost.com seems to be heavy commercial site, which I would prefer not to link to

Comment (Arsnael) [reply server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/DomainQuotaRoutesTest.java]: Didn't even think about that oO just did a silly on the fly translation... Will check that :)

Comment (Arsnael) [reply server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/DomainQuotaRoutesTest.java]: It's tests after, shouldn't probably use existing domain names at all

**Comment by Arsnael** [comment]
```
[ERROR]   CassandraMailRepositoryTest.storingAndRemovingMessagesConcurrentlyShouldLeadToConsistentResult » Execution
```
test this please

**Comment by chibenwa** [comment]
`MemorySpamAssassinContractTest.expungingSpamMessageShouldNotImpactSpamAssassinState » ConditionTimeout`

test this please

**Comment by chibenwa** [comment]
Merged

File: server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/DomainQuotaRoutesNoVirtualHostingTest.java [hunk 2]
```diff
@@ -44,7 +44,7 @@
 class DomainQuotaRoutesNoVirtualHostingTest {
 
     private static final String QUOTA_DOMAINS = "/quota/domains";
-    private static final String FOUND_COM = "found.com";
+    private static final String FOUND_LOCAL = "found.local";
```
Comment (mbaechler) [inline server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/DomainQuotaRoutesNoVirtualHostingTest.java]: I'm late to commentthis review but `.local` doesn't seem wise as it's never considered an internet domain (see https://en.wikipedia.org/wiki/.local )

Comment (rouazana) [reply server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/DomainQuotaRoutesNoVirtualHostingTest.java]: no a big issue imho, but if needed we can use .test instead.

**Pull request closed without merging**